com.mucommander.Mucommander.yml

```diff
@@ -0,0 +1,50 @@
+app-id: com.mucommander.Mucommander
+branch: '1.0'
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: run.sh
+finish-args:
+  - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+  - --env=JAVA_HOME=/app/jre # stops picking up local user JRE  
+  - --socket=x11
+  - --share=network
+  - --share=ipc
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=host
```

Comment:
this is an extremely broad filesystem permission

Reply:
Yeah but it's a file manager